ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataSciencePipelines.resource

```diff
@@ -1,17 +1,16 @@
 *** Settings ***
-Documentation       Test suite for OpenShift Pipeline
-Resource            ../../../RHOSi.resource
-Resource            ../../../ODS.robot
-Resource            ../../../Common.robot
-Resource            ../../../Page/ODH/ODHDashboard/ODHDashboard.robot
+Documentation       Collection of keywords to interact with Data Science Pipelines
```

Comment:
could you keep this file in its original path?
`ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource` 

The ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject I think should contain only keywords interacting with the DS Project UI directly. Like done for model serving. We cpuld discuss about refactor the folder structure, but I think it's better to keep consistency for the moment

Reply:
Let's address this in other PR
we have also now the following:
* ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataSciencePipelines.resource
* ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Pipelines.resource

Reply:
Done